diff --git a/node_modules/@react-navigation/native-stack/lib/module/views/NativeStackView.native.js b/node_modules/@react-navigation/native-stack/lib/module/views/NativeStackView.native.js
index c342e90..2228551 100644
--- a/node_modules/@react-navigation/native-stack/lib/module/views/NativeStackView.native.js
+++ b/node_modules/@react-navigation/native-stack/lib/module/views/NativeStackView.native.js
@@ -144,7 +144,7 @@ const SceneView = ({
   // Debounce the header height updates to avoid excessive re-renders
   debounce(setHeaderHeight, 100), []);
   const hasCustomHeader = header != null;
-  const usesNewAndroidHeaderHeightImplementation = 'usesNewAndroidHeaderHeightImplementation' in compatibilityFlags && compatibilityFlags['usesNewAndroidHeaderHeightImplementation'] === true;
+  const usesNewAndroidHeaderHeightImplementation = compatibilityFlags && 'usesNewAndroidHeaderHeightImplementation' in compatibilityFlags && compatibilityFlags['usesNewAndroidHeaderHeightImplementation'] === true;
   let headerHeightCorrectionOffset = 0;
   if (Platform.OS === 'android' && !hasCustomHeader && !usesNewAndroidHeaderHeightImplementation) {
     const statusBarHeight = StatusBar.currentHeight ?? 0;
